app/models/version.rb

```diff
@@ -28,6 +28,10 @@ def self.previous_production
   end
 
   def self.current_preview
+    if Version.current_production && Version.preview.newest.first.created_at < Version.current_production.created_at
+      return Version.current_production
```

Comment:
This adds confusion.  The method is named `current_preview`, so I would expect it to always return a `preview` or perhaps `nil`

Reply:
The way it functioned previously was it would return the latest preview version whether if that version was a duplicate of the production or not. I figured that by refactoring it to return the production version when there are no new preview versions, we would still retain how `current_preview` was being used. This is necessary as `current_preview` returns the `updated_at` date we need to determine which uploads are being used for generating the new preview version. 

I could move that logic to `since_last_preview_version` in `upload.rb` instead of leaving it in `version.rb`. @jholton what are your thoughts?